lib/client/js/ui/controls/Menu.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * Insert description of Menu class
+ *
+ * @private
```

Comment:
Question: We have the Control class labeled as Private in its JSDoc, and these two classes seem to be structured in very similar ways. What makes them different in this sense? Or should neither of them be labeled as Private?